meshroom/nodes/aliceVision/PanoramaPostProcessing.py

```diff
@@ -32,14 +32,32 @@ class PanoramaPostProcessing(desc.CommandLineNode):
             uid=[0],
         ),
         desc.ChoiceParam(
-                name='outputColorSpace',
-                label='Output Color Space',
-                description='Allows you to choose the color space of the output image.',
-                value='Linear',
-                values=['sRGB', 'Linear', 'ACES2065-1', 'ACEScg'],
-                exclusive=True,
-                uid=[0],
+            name='outputColorSpace',
+            label='Output Color Space',
+            description='Allows you to choose the color space of the output image.',
+            value='Linear',
+            values=['sRGB', 'Linear', 'ACES2065-1', 'ACEScg'],
+            exclusive=True,
+            uid=[0],
         ),
+        desc.ChoiceParam(
+            name='compressionMethod',
+            label='Compression Method',
+            description='Compression method for output image.',
+            value='Auto',
+            values=['None', 'Auto', 'RLE', 'ZIP', 'ZIPS', 'PIZ', 'PXR24', 'B44', 'B44A', 'DWAA', 'DWAB'],
+            exclusive=True,
+            uid=[0],
+        ),
+        desc.IntParam(
+            name='compressionLevel',
+            label='Compression Level',
+            description='Level of compression relying on the selected compression method.',
+            value=0,
+            range=(0, 200, 1),
```

Comment:
Is there a reason to have a max  > 100 ?

Reply:
Yes, compression level for dwaa and dwab methods is not limited to 100. It can be as high as the user wants. However, level higher than 200 lead to bad quality images.